### PR TITLE
Basic upload page

### PR DESCRIPTION
**Pull request by wmalik-mux**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,16 @@
...
     "@mux/mux-player-react": "^1.8.0",
+    "@mux/mux-uploader-react": "^0.1.0-canary.218-5856b1b",
     "next": "^13.2.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "18.15.11",
+    "@types/react": "18.0.33",
+    "typescript": "5.0.3"
```
Comment (refactornator) [inline package.json]: The `typescript` dependency should be removed and `npm install` should be re-run.

Comment (wmalik-mux) [reply package.json]: 👍 Removed it and uninstalled.

**Review by refactornator — APPROVED** [review]
It works! 🎉

**Comment by wmalik-mux** [comment]
Going to keep this branch open since @davekiss wanted to try something earlier.

**Pull request merged**